### DISCUSSION TITLE
Changing the first day of the week

### DISCUSSION THREAD

**Discussion by TheBearDidLady** [TechbeeAT/jtxBoard#235] [Q&A] [answered]
I've had a solid dig around in the UI, and searched the content in here, and I can't find a way to do something I'd really like to do. Either it's not currently an available piece of functionality, or it's inherited from my DavX and/or server settings.

What I want to do is change the starting day of the week from Sunday to Monday anywhere a calendar appears, including inside the date picker.

For me, Monday is the first day of the week, and it always throws me when software assumes Sunday is.

Any thoughts? Have I missed something in the UI or the online discussions?

**Comment by patrickunterwegs** [comment, marked as answer]
Hey @TheBearDidLady !
That's indeed a bit strange in Android. Basically the date picker is prepared to use Monday or Sunday as first day of the week. There seems to be no dedicated system setting in Android to change this, but the setting is implicitly set through the language settings. 
For example when I set the system language to "English (US)", the first day of the week would be Sunday. 
When I set the system language to "English (Austria)", the first day of the week would be Monday.
Remark: The in-app setting for the language is set to system default in this case! (The in-app language picker just changes the language, it is not so clear what country Android would choose in the background)

Is this an option for you to change the language setting with a specific country?
I can also think of adding a setting specifically to change the first day of the week, but there is another option, I'd rather avoid it (for the sake of keeping settings simple) 😉

**Reply by patrickunterwegs** [reply]
Hey @TheBearDidLady , if the problem is solved with my answer, could you mark it as solved with the "Mark as answer" button?

**Reply by TheBearDidLady** [reply]
Thank you for getting back to me with such a solid, complete answer so quickly. Sorry I didn't flag it as answered until now. It's been a busy week or two.

I can confirm that changing my language settings to English (Australia) does the trick for me. Fortunately, I was able to do that on an app-specific basis, so it hasn't affected my system level language settings. It was a bit of a weird design decision for Android to behave this way, but it's by no means the only example of such a decision. I'm just very happy you gave me an easy workaround.

**Reply by patrickunterwegs** [reply]
Thanks for the update, I'm glad it could be solved by changing the language settings! 👍🙂
